utils/make-sqlite.py

```diff
@@ -37,7 +37,12 @@
 c.execute('''DROP TABLE IF EXISTS targets''')
 c.execute('''CREATE TABLE targets
              (host TEXT,
+              platform_id INTEGER,
               ruleset_id INTEGER)''')
+c.execute('''DROP TABLE IF EXISTS platforms''')
+c.execute('''CREATE TABLE platforms
```

Comment:
It's weird to add new code that depends on SQLite when we are planning to remove SQLite altogether. Can you do this in-memory in trivial-validate.py instead?